### ISSUE TITLE
Cog PSY 73800 students post here to say hello

### ISSUE THREAD

**Issue by CrumpLab** [CrumpLab/psyc73800#1]
Leave a comment here to let me know you made your account. Then, if you run into issues with the process of getting your R Markdown website working, create a new issue here and ask your question. I will try to respond ASAP. If you know the answer to someone else's question, please help out if you have the time and leave a comment.

**Comment by ghost** [comment]
Account made.

**Comment by ChloeBrittenham** [comment]
Made an account!

-Chloe

**Comment by selmaliah** [comment]
Account made

Sapir Elimaliah

**Comment by dreyess** [comment]
I made an account. 

- Daisy 

_____________ 

When I click "build website" on RStudio, the following appears, preventing me from making my website: 

==> rmarkdown::render_site(encoding = 'UTF-8')

Error in rmarkdown::render_site(encoding = "UTF-8") : 
  No site generator found.
Execution halted

Exited with status 1.


Any help will be greatly appreciated.

**Comment by sjacobson1112** [comment]
Hello! I made an account.
Sarah

**Comment by kellycotton** [comment]
Made an account! 
-Kelly

**Comment by hyeincho8804** [comment]
Hi, it turns out I had an account. 
I will try to make a website now! 
Hyein

**Comment by Chris-Hoffman** [comment]
Mission accomplished, Professor!

**Comment by jmgo1935** [comment]
I made an account!
-Josephine O'Malley

**Comment by mhorger** [comment]
Got it!

**Comment by Chris-Hoffman** [comment]
I'm unfortunately running into some problems creating my website! I've followed the steps but keep getting: Error in match.arg(highlight, html_highlighters()) : 
  'arg' should be one of “default”, “tango”, “pygments”, “kate”, “monochrome”, “espresso”, “zenburn”, “haddock”, “textmate”

For others having problems, I found this YouTube video helpful (Until the end, when I ran into problems!)
https://www.youtube.com/watch?v=QYr06j_JMdo

**Comment by hyeincho8804** [comment]
Hi, I am also running into some issues, where I made all the changes to the template in R Studio but can't have it published online. I think there are some problems with the sync, but I am definitely stuck in the last few steps...

**Comment by CrumpLab** [comment]
Daisy - Have you installed the rmarkdown package? Also, have you changed or renamed the _site.yml file?

Chris - Not sure about this one either, have you installed the rmarkdown package? Thanks for posting the link

Hyein - Have you downloaded github desktop? Here are some steps:

1. Download github desktop
2. Connect it to your github account
3. Clone your online repository with the website code to your local computer
4. Edit the files locally in R-studio, compile new website
5. After you have made changes, you will need to push them back up to github. This is a two-step process. In step 1 you "commit" the changes (you will be asked write a short note describing the changes you made), and then you need to "sync" your local repository with the one on github...

I'm going to make a short screencast about these steps and post for everyone this morning.

**Comment by CrumpLab** [comment]
Here's a link to a youtube I just made walking through the process of getting an R Markdown website up an running on github.

[https://www.youtube.com/watch?v=pxUunAh8B6U&feature=youtu.be](https://www.youtube.com/watch?v=pxUunAh8B6U&feature=youtu.be)

**Comment by genejerskey** [comment]
Hello, world!

**Comment by sjacobson1112** [comment]
I'm ran into a problem installing rmarkdown because there were errors in the associated package downloads. I found a suggestion online to download them as type="binary." After I did this it worked, but I am curious why this was necessary and what the binary designation does?
Thanks!

**Comment by ghost** [comment]
@sjacobson1112 same here... updated High Sierra and just got it to work

**Comment by DanaRSc** [comment]
Acount made.

Dana

**Comment by LennayC** [comment]
Account also made!

Lennay

**Comment by StephanieSorage** [comment]
Account made!

- Stephanie

**Comment by DanaRSc** [comment]
**Problem solved - it was because I didn't extract the zip folder.

Hi there,
Unfortunately I have the same error message Daisy had - when I clicked on "build website" I got:
==> rmarkdown::render_site(encoding = 'UTF-8')

Error in rmarkdown::render_site(encoding = "UTF-8") : 
  No site generator found.
Execution halted

Exited with status 1.

I did make sure I have rmarkdown installed, and I didn't change the name of anything - I followed the instructions on the video.

Any help would be much appreciated.
Thanks!
Dana
